lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -43,6 +43,8 @@ function collectImageElementInfo() {
       naturalHeight: element.naturalHeight,
       isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
+      usesObjectFit: window.getComputedStyle(element).getPropertyValue('object-fit') === 'cover'
```

Comment:
nit: let's try to make only one call to `window.getComputedStyle` and clean this line up a bit